core-services/release-controller/_releases/release-ocp-4.12-ci.json

```diff
@@ -71,6 +70,7 @@
           "ironic-inspector",
           "ironic-ipa-downloader",
           "ironic-machine-os-downloader",
+          "ironic-static-ip-manager",
```

Comment:
I think the addition of this line is not right here

Reply:
Instead, the line above should be removed here. There are two configs in this file, which should both be changed in the same way. (remove `ironic-machine-os-downloader` from the excludeTags)